_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -5983,6 +5986,11 @@
 /obj/effect/turf_decal/stripes/line{
 	dir = 1
 	},
+/obj/structure/cable{
+	d1 = 1;
```

Comment:
`d1 = *` & `d2 = *` aren't needed and should be removed please.

Reply:
Admittedly I don't actually know what you mean by this since I'm not really at all knowledgeable when it comes to code. Could you elaborate in terms of a simpleton such as myself?

Reply:
d1 & d2 are direction vars for wires that determines where they connect, except they aren't needed since the system now generates those variables based off the wire's icon_state; ergo it's just leftover code.

Reply:
Alrighty, I'll take a look in that later and see what I can do. Thanks for the headsup.

Reply:
@ShizCalev Come to think of it, don't all of the maps have d1 and d2s? Not really sure how I should go about removing them from delta when they just appear to be leftover code, as you said. Any ideas how I could go about doing this so I can get these updates merged?

Reply:
It was fixed on the other maps with #32427

If you have notepad++, open delta up in it, go to replace (control f), check off regular expression at the bottom, put `d1 = .*\r\n` in find and replace with blank, then do `d2 = *\r\n` to remove them all.